crates/bevy_window/src/window.rs

```diff
@@ -184,6 +184,14 @@ pub struct Window {
     ///
     /// - iOS / Android / Web: Unsupported.
     pub ime_position: Vec2,
+    /// Sets a specific theme for the window.
+    ///
+    /// If `None` is provided, the window will use the system theme.
+    ///
+    /// ## Platform-specific
+    ///
+    /// - iOS / Android / Web / x11: Unsupported.
+    pub preferred_theme: Option<WindowTheme>,
```

Comment:
This field is actually unused in this implementation. Methods added in this PR modify `window.internal.preferred_theme`, not `window.preferred_theme`, and only the internal fields are read. So this does literally nothing.

I suggest removing `window.internal.preferred_theme` and remove the `set_window_theme` and `window_theme` methods on `Window` and directly react to changes to this field instead of the internal field.

Reply:
Thank you very much for your input, I'll push a PR shortly